openwisp_monitoring/settings.py

```diff
@@ -1,7 +1,30 @@
 from django.conf import settings
+from openwisp_notifications.types import register_notification_type
 
 AUTO_GRAPHS = getattr(
     settings,
     'OPENWISP_MONITORING_AUTO_GRAPHS',
     ('traffic', 'wifi_clients', 'uptime', 'packet_loss', 'rtt',),
 )
+
+register_notification_type(
+    'threshold crossed',
+    {
+        'name': 'Monitoring Alert',
+        'verb': 'crossed threshold limit',
+        'level': 'warning',
+        'email_subject': '[{site}] {notification.level} - {notification.target} has {notification.verb}',
```

Comment:
shouldn't this be ``site.name``?

Reply:
https://github.com/django/django/blob/06c8565a4650b359bdfa59f9707eaa0d1dfd7223/django/contrib/sites/models.py#L96

It returns `site.domain`, are we not interested in that?

Reply:
use `site.name`. the domain is for machines, the name is for humans. We must put effort in designing the software for humans, not machines.